src/boot/memory.c

```diff
@@ -352,10 +361,17 @@ void *load_segment(s32 segment, u8 *srcStart, u8 *srcEnd, u32 side, u8 *bssStart
 void *load_to_fixed_pool_addr(u8 *destAddr, u8 *srcStart, u8 *srcEnd) {
     void *dest = NULL;
     u32 srcSize = ALIGN16(srcEnd - srcStart);
-    u32 destSize = ALIGN16((u8 *) sPoolListHeadR - destAddr);
+    u32 destSize = ALIGN16((u8 *) RAM_END - destAddr);
```

Comment:
is this assuming that main pool ends at end of ram? i think there might be a better variable to use here although not sure

Reply:
`load_to_fixed_pool_addr` is loading at fixed address in the end of RAM.